src/mds/Mutation.h

```diff
@@ -407,6 +411,8 @@ struct MDRequestImpl : public MutationImpl {
 
   // TrackedOp stuff
   typedef boost::intrusive_ptr<MDRequestImpl> Ref;
+  set<Ref> batch_reqs;
+  Ref batch_head;
```

Comment:
I don't see an actual reason for this `batch_head` reference. It seems extraneous.